apps/www/_alternatives/supabase-vs-firebase.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Supabase vs Firebase
+description: Supabase vs Firebase
+author: ant_wilson
+image: all-pull-together/all-pull-together-thumb.png
+thumb: all-pull-together/all-pull-together-thumb.png
+tags:
+  - community
+  - case-study
+  - q-and-a
+date: '2022-05-26'
+toc_depth: 3
+---
+
+## What is Firebase?
+
+Now owned by Google, Firebase is a collection of tools aimed at mobile and web developers. At its core is the Firestore database.
+
+Firestore allows you to store “documents”. These are collections of key:value pairs where the value can be another sub-document. Document based storage is perfect for unstructured data, since two documents in a collection do not necessarily need to have the same structure.
+
+Firebase also offers other things that web developers find useful like an auth service for user management, and wrappers for other Google services such as Cloud Functions, and File Storage.
+
+## What is Supabase?
+
+Supabase is an open source firebase alternative, but instead of being built around a document-based datastore, Supabase offers a relational database management system called PostgreSQL. This comes with a few advantages:
```

Comment:
```suggestion
Supabase is an open source Firebase alternative, but instead of being built around a document-based datastore, Supabase offers a relational database management system called PostgreSQL. This comes with a few advantages:
```